proposed/http-middleware/middleware-meta.md

```diff
@@ -268,19 +287,38 @@ The `DelegateInterface` defines a single method that accepts a request and
 returns a response. The delegate interface must be implemented by any middleware
 dispatcher that uses middleware implementing `MiddlewareInterface`.
 
+#### Why the term "delegate"?
+
+The term "delegate" means something designated to act for or represent another.
+In terms of middleware design, a delegate is called upon by middleware when the
+middleware is unable to handle the request itself; the delegate then processes
+the request _for the original middleware_ in order to return a response.
```

Comment:
BIG :+1: Many, many thanks and I love that this avoid the _next middleware component_ wording.